src/lib/utils/cx.ts

```diff
@@ -0,0 +1,3 @@
+export function cx(cssClasses?: string | string[]) {
+  return Array.isArray(cssClasses) ? cssClasses.join(' ') : cssClasses;
+}
```

Comment:
This is only useful until we get rid of `string[]` as a possible type for `cssClasses`.